content/lessons/done/translations.ts

```diff
@@ -2,10 +2,14 @@ export const translations = {
   done: {
     en: {
       title: 'Done',
```

Comment:
Since we are fixing the way the title is displayed. Let's also update the text displayed to match the Figma file.
```suggestion
      title: 'You did it!!!',
```